.github/workflows/tag.yml

```diff
@@ -0,0 +1,28 @@
+name: flask-rebar Pull Request Tests
+
+on:
+  release:
+    types:
+    - created
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pep517
+      run: |
+        python -m pip install pep517 --user
+    - name: Build a binary wheel and a source tarball
+      run: |
+        python -m pep517.build .
```

Comment:
I don't know about PEP517, can you explain a bit the choice vs others like poetry, flit and basic setup.py? Since the project isn't using new format from PEP517/518 I don't really see the benefit.

Reply:
This still uses setup.py.

Python 517 just specifies the build tools section of the pyproject.toml, which tells everyone how the package should be built into an sdist and/or a wheel.  So if we ever move to flit/poetry this build method will still work.